hw/dv/verilator/cpp/verilator_memutil.cc

```diff
@@ -71,6 +71,8 @@ static void PrintHelp() {
                "  Initialize the RAM with FILE (elf/vmem)\n\n"
                "-f|--flashinit=FILE\n"
                "  Initialize the FLASH with FILE (elf/vmem)\n\n"
+               "-o|--otpinit=FILE\n"
+               "  Initialize the OTP with FILE (elf/vmem)\n\n"
```

Comment:
The rominit/raminit arguments are backwards-compat only, no need to add a new one. --meminit is sufficient. You can remove these arguments.

Reply:
Just to clarify, do you mean removing all three of them?

Reply:
I just removed `otpinit` for now.

Reply:
Yes, that's what I meant, sorry for the confusion.